layouts/partials/navbar.html

```diff
@@ -45,11 +46,31 @@
           </a>
 
           <div class="navbar-dropdown is-hidden-touch">
-            {{- range $docs }}
+            {{ range $docs }}
             <a class="navbar-item" href="{{ .URL }}">
               {{ .Title }}
             </a>
-            {{- end }} <!-- range $docs -->
+            {{ end }} <!-- range $docs -->
```

Comment:
Right now the `Docs` ion Navbar links to `/docs/` (a blank page), can it go to `/docs/concepts/overview/`?

Reply:
Good catch